daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -129,9 +135,7 @@ def DistroSpecific(g):
 
     remove_azure_agents(g)
 
-    g.write(
-        '/etc/apt/sources.list.d/partner.list',
-        partner_list.format(ubu_release=ubu_release))
+    add_gce_repositories(g, ubu_release)
```

Comment:
is this intended, to have both `release` and `ubu_release` params?

Reply:
Consolidated `release` and `ubu_release` to use `ubuntu_release`, which is the name of the metadata value passed in by daisy.